components/contract-components/tables/deployed-contracts.tsx

```diff
@@ -86,10 +86,26 @@ export const DeployedContracts: React.FC<DeployedContractsProps> = ({
   contractListQuery,
   limit = 10,
 }) => {
+  const [showMoreLimit, setShowMoreLimit] = useState(limit);
```

Comment:
@MananTank I like what you did with the design!

I think this commit introduced a regression bug though. I changed around how Show More works because the filter needs to be able to filter all of your contracts (even the ones that are not on the current page). Going back to this method of Show More will only filter contracts on the current page. Is there a good reason to revert back to this method of Show More?

Reply:
@mykcryptodev I've taken care of it - `chainIdsWithDeployments` is calculated with `contractListQuery.data` and not the sliced data

Reply:
my bad - you are right. we can't have show more the way list is filtered

Reply:
@MananTank Im on it - I can open a PR -- logged issue #1204

Reply:
I pushed a fix - https://github.com/thirdweb-dev/dashboard/pull/1205